language/src/speech_processor.py

```diff
@@ -391,27 +398,198 @@ async def text_to_speech(
         # )
         return new_audio_content
 
```

Comment:
Add a new line here as whenever a new class is present, two new lines should be there.